src/components/version-language-switcher/version-language-switcher.css

```diff
@@ -28,4 +28,37 @@
     margin-top: 4px;
     display: flex; /* if needed to align items horizontally */
     align-items: center; /* vertically align content */
-}
\ No newline at end of file
+}
+
+@media (max-width: 623px) {
+    [slot="version"] {
+        display: flex; /* if needed to align items horizontally */
+        flex-direction: column;
+        align-items: flex-start;
+        gap: 2px;
+        margin-top: 0;
+
+    }
+}
+
+@media (max-width: 623px) {
+    [slot="version"] {
+        display: flex; /* if needed to align items horizontally */
+        flex-direction: column;
+        align-items: flex-start;
+        gap: 2px;
+        margin-top: 0;
+
+    }
+}
+
+@media (max-width: 623px) {
```

Comment:
It looks like you repeated the same media query three times. You can delete the second two.